main.go

```diff
@@ -17,16 +21,40 @@ func main() {
 	flag.BoolVar(&version, "version", false, "print version and exit")
 	flag.Parse()
 
+	if version {
+		fmt.Println("Browserpass host app version:", VERSION)
+		os.Exit(0)
+	}
+
+	openbsd.Pledge("stdio rpath wpath proc exec")
+
 	log.SetFormatter(&log.TextFormatter{FullTimestamp: true})
 	if verbose {
 		log.SetLevel(log.DebugLevel)
 	}
 
-	if version {
-		log.Info("Browserpass host app version: ", VERSION)
-		os.Exit(0)
+	logFilePath, err := getLogFilePath()
```

Comment:
Why not use syslog? That way you don't need to care about where the file lives, and the user can configure all the normal logging things without needing to worry about what browserpass is doing. Logging to a file seems like a good fallback option, but it seems strange to use it as the first choice.